test/galore/configs/llama_1b.json

```diff
@@ -0,0 +1,18 @@
+{
+  "architectures": ["LLaMAForCausalLM"],
+  "bos_token_id": 0,
+  "eos_token_id": 1,
+  "hidden_act": "silu",
+  "hidden_size": 2048,
+  "intermediate_size": 5461,
+  "initializer_range": 0.02,
+  "max_sequence_length": 1024,
+  "model_type": "llama",
+  "num_attention_heads": 32,
+  "num_hidden_layers": 24,
+  "pad_token_id": -1,
+  "rms_norm_eps": 1e-6,
+  "transformers_version": "4.28.1",
+  "use_cache": true,
+  "vocab_size": 32000
+}
```

Comment:
just a heads up we can't merge binary files in the git repo so this includes html, pt files and most of the json files that were merged in